README.md

```diff
@@ -44,6 +44,21 @@ function names_. This hinders debugging and confuses automatic crash stack conso
 npm install @bloomberg/pasta-sourcemaps
 ```
 
+## Development
```

Comment:
Added this section in response to losing the project-specifics from CONTRIBUTING.md

Reply:
This is good. I wonder if it makes sense to move this section further down, just before "Contributions"? I expect more people would be interested in how to use this, rather than how to develop / contribute.